package.json

```diff
@@ -25,7 +25,7 @@
   ],
   "dependencies": {
     "queue-microtask": "^1.1.2",
-    "readable-stream": "^3.6.0"
+    "readable-stream": "^3.6.0|^4.0.0"
```

Comment:
Why is it necessary to keep v3?

Reply:
No, I just wanted to avoid a breaking change.

Reply:
Seems that v4 only works on Node 12 and up—we still CI on Node 10. Maybe the choice is the right solution for now indeed then.

Reply:
Dropped 10 for #300 so might as well go with 4.